Makefile

```diff
@@ -11,16 +40,9 @@ add: pyproject.toml
 uninstall: pyproject.toml
 	poetry remove $(package)
 
```

Comment:
you could add a command called `setup` that does everything a dev needs to run after pulling the code, something like:
```
setup: up install migrate
```